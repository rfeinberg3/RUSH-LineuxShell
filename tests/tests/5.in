path /autograder/source/bin
p5.sh & p5.sh & p4.sh
exit
