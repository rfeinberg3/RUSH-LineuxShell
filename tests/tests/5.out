rush> rush> dog
panther
panther
rush> 